sc2reader/data/attributes.json

```diff
@@ -715,7 +715,8 @@
         "Tych": "Tychus",
         "Vora": "Vorazun",
         "Zaga": "Zagara",
-        "Zera": "Zeratul"
+        "Zera": "Zeratul",
+        "Stet": "Stetmann"
```

Comment:
Can't we keep the keys in alphabetical order to make it easier to spot duplicate keys?